packages/server/lib/routes.ts

```diff
@@ -272,13 +278,13 @@ web.route('/api/v1/users/:userId/suspend').post(webAuth, userController.suspend.
 web.route('/api/v1/sync').get(webAuth, syncController.getSyncsByParams.bind(syncController));
 web.route('/api/v1/sync/command').post(webAuth, syncController.syncCommand.bind(syncController));
 web.route('/api/v1/syncs').get(webAuth, syncController.getSyncs.bind(syncController));
-web.route('/api/v1/sync/:syncId/frequency').put(webAuth, syncController.updateFrequency.bind(syncController));
 web.route('/api/v1/flows').get(webAuth, flowController.getFlows.bind(syncController));
-web.route('/api/v1/flow/pre-built/deploy').post(webAuth, postPreBuiltDeploy);
-web.route('/api/v1/flow/pre-built/upgrade').put(webAuth, putUpgradePreBuilt);
+web.route('/api/v1/flows/pre-built/deploy').post(webAuth, postPreBuiltDeploy);
+web.route('/api/v1/flows/pre-built/upgrade').put(webAuth, putUpgradePreBuilt);
```

Comment:
I feel bad that we now have `flows` when we sometimes refer to them `scripts`. It is probably confusing to have different terms floating around for how we refer to `syncs` and `actions` etc

Reply:
yep, that's unfortunate hopefully we'll settle on one wording